mobile/src/main/java/com/example/android/uamp/MediaNotificationManager.java

```diff
@@ -296,6 +303,12 @@ private Notification createNotification() {
                     R.drawable.ic_default_art);
             }
         }
+        /**
```

Comment:
You can probably remove this comment, or change it to a single line, non-javadoc comment, please.